client/src/pages/CreateTeam.js

```diff
@@ -158,31 +258,31 @@ const CreateTeam = () => {
               <div className="w-1/3 mx-2 py-5">
                 <form>
                   <label htmlFor="input3">Choice 7</label>
-                  <input
-                    type="text"
-                    id="c7"
-                    className="border rounded py-2 px-3 w-full"
+                  <Select
+                    options={projectOptions}
+                    onChange={handleProject7}
+                    className=" py-2 w-full"
                   />
                 </form>
               </div>
               <div className="w-1/3 mx-2 py-5">
                 <form>
                   <label htmlFor="input3">Choice 8</label>
-                  <input
-                    type="text"
-                    id="c8"
-                    className="border rounded py-2 px-3 w-full"
+                  <Select
+                    options={projectOptions}
+                    onChange={handleProject8}
+                    className=" py-2 w-full"
                   />
                 </form>
               </div>
 
               <div className="w-1/3 mx-2 py-5">
                 <form>
                   <label htmlFor="input3">Choice 9</label>
-                  <input
-                    type="text"
-                    id="c9"
-                    className="border rounded py-2 px-3 w-full"
+                  <Select
```

Comment:
Not necessary for now, but if we have time to clean it up more later, it would be really nice to have it where a project disappears once it has been selected in one of the boxes! So if you select "Project 1" in box 1, it is no longer an option when you are making your selection in Box 2.